src/io/linux.zig

```diff
@@ -992,69 +992,117 @@ pub const IO = struct {
     ) !os.fd_t {
         assert(relative_path.len > 0);
         assert(size % constants.sector_size == 0);
```

Comment:
Can we assert that `size` is never 0?

Reply:
It appears we can't, since the AOF uses a size of 0.

Reply:
hmmm, that means that the file might not even be `superblock_zone_size` large...

Reply:
Oh, the real issue here is that the AOF doesn't support block devices.